### PR TITLE
[native] Move static functions from PrestoTask to Util.

### PR DESCRIPTION
**Pull request by amitkdutta**
## Description
Minor refactor to move static tasks from PrestoTask struct to Util.

## Motivation and Context
We are trying to make PrestoTask a class instead of a struct for better encapsulation. Moving static methods out of the struct makes this easier. Besides the functions we are moving aren't a good fit for PrestoTask.

## Impact
No impact.

## Test Plan
Existing unit tests

### CODE REVIEW

File: presto-native-execution/presto_cpp/main/common/Utils.h [hunk 1]
```diff
@@ -31,4 +31,9 @@ std::shared_ptr<folly::SSLContext> createSSLContext(
     const std::string& clientCertAndKeyPath,
     const std::string& ciphers);
 
+/// Returns process-wide CPU time in nanoseconds.
+long getProcessCpuTime();
```
Comment (xiaoxmeng) [inline presto-native-execution/presto_cpp/main/common/Utils.h:35]:
```
nit: s/getProcessCpuTime/getProcessCpuTimeNs/?
```

File: presto-native-execution/presto_cpp/main/common/Utils.h [hunk 2]
```diff
@@ -31,4 +31,9 @@ std::shared_ptr<folly::SSLContext> createSSLContext(
...
+
+std::string taskNumbersToString(const std::array<size_t, 5>& taskNumbers);
```
Comment (xiaoxmeng) [inline presto-native-execution/presto_cpp/main/common/Utils.h:37]:
```
const std::array<size_t, 5>& taskNumbers
```

File: presto-native-execution/presto_cpp/main/TaskResource.cpp [hunk 3]
```diff
@@ -220,7 +221,7 @@ proxygen::RequestHandler* TaskResource::createOrUpdateTaskImpl(
         folly::via(
             httpSrvCpuExecutor_,
             [this, &body, taskId, createOrUpdateFunc]() {
-              const auto startProcessCpuTime = PrestoTask::getProcessCpuTime();
+              const auto startProcessCpuTime = util::getProcessCpuTime();
```
Comment (xiaoxmeng) [inline presto-native-execution/presto_cpp/main/TaskResource.cpp:224]:
```
NYC: s/startProcessCpuTime/startProcessCpuTimeNs/
```

File: presto-native-execution/presto_cpp/main/PrestoTask.cpp [hunk 4]
```diff
@@ -775,27 +763,6 @@ void PrestoTask::updateExecutionInfoLocked(
       prestoTaskStats);
 }
 
-/*static*/ std::string PrestoTask::taskNumbersToString(
```
Comment (xiaoxmeng) [inline presto-native-execution/presto_cpp/main/PrestoTask.cpp:778]: We might want to keep this as static method of PrestoTask as the states are specific to PrestoTask internal

s/taskNumbersToString/taskStatesToString/

**Review by xiaoxmeng — COMMENTED** [review]
@amitkdutta thanks for the improvement!

File: presto-native-execution/presto_cpp/main/common/Utils.h [hunk 5]
```diff
@@ -31,4 +31,9 @@ std::shared_ptr<folly::SSLContext> createSSLContext(
     const std::string& clientCertAndKeyPath,
     const std::string& ciphers);
 
+/// Returns process-wide CPU time in nanoseconds.
```
Comment (mbasmanova) [inline presto-native-execution/presto_cpp/main/common/Utils.h:34]: It would be nice to avoid files / classes named Util as they tend to become a "dumping ground" for lots of unrelated code. It would be better to come with more specific names.

**Comment by amitkdutta** [comment]
Closing this in favor of https://github.com/prestodb/presto/pull/22662
